tests/pathfinding/test_api.py

```diff
@@ -266,6 +271,44 @@ def test_get_paths(api_url: str, addresses: List[Address], token_network_model:
         assert response.status_code == 404
         assert response.json()["error_code"] == exceptions.NoRouteFound.error_code
 
+    initiator_address = to_checksum_address(addresses[0])
```

Comment:
This part of the test is totally separate from the previous one and sets up its own helpers, so I think it would be easier to read in a separate test function.